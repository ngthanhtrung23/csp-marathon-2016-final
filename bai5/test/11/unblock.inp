.12344
.12355
002...
6777.8
6....8
99AA.8
