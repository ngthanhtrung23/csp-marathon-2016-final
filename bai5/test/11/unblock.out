21
8 U 1
A R 2
9 R 1
6 D 1
7 L 1
3 D 4
4 L 1
5 L 1
8 U 2
7 R 3
6 U 1
9 L 1
2 D 3
0 R 3
2 U 3
6 U 2
7 L 3
3 U 1
A L 1
8 D 3
0 R 1

